### DISCUSSION TITLE
List of words not encountered by the tool

### DISCUSSION THREAD

**Discussion by PatrickHammond66** [facelessuser/pyspelling#140] [Q&A] [answered]
Hello there,

I want to output words that we have defined in our word dictionaries, that no longer appear in the text that we are running spellcheck on.

It is likely that as content changes over time, and as we improve regexs for filtering out junk, some of the words defined early on will no longer be encountered by the tool. Is it possible to output a list of words that the tool has defined in dictionaries, but do not appear in the text?

Currently my wordlists are quite long, so I'm attempting to minimize them!

Thanks,

Patrick

**Comment by facelessuser** [comment, marked as answer]
@PatrickHammond66 the word list is compiled and given to either Aspell or Hunspell. Nothing more is done with it than that.
